From 9c91db847318dfd0de4a65a4fc729d41fe502836 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Wed, 20 Mar 2024 16:47:01 +0200
Subject: [PATCH] [FSW-4100] fs1: rename arc cpus

In FS1, unlike previous gaudi devices, the PSOC ARCs are named, not
numbered.
Rename the address definitions to match FS1 naming convention

Change-Id: I5019fa7a5fdbd68e508ef6264c1e285e5ce00c67
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/fs1/arc/common.h b/arch/fs1/arc/common.h
index 57f6157..766dfee 100644
--- a/arch/fs1/arc/common.h
+++ b/arch/fs1/arc/common.h
@@ -22,21 +22,21 @@
 #define HL_CFG_MEM_SIZE			SZ_64M
 #define HL_PCIE_FW_SRAM_BASE_FULL	0x300007FF8B80000ULL
 
-#define ARC0_MEM_SIZE			(ARC0_ICCM_SIZE + ARC0_DCCM_SIZE)
-#define ARC1_MEM_SIZE			(ARC1_ICCM_SIZE + ARC1_DCCM_SIZE)
+#define ARC_PID_MEM_SIZE		(ARC_PID_ICCM_SIZE + ARC_PID_DCCM_SIZE)
+#define ARC_MNG_MEM_SIZE		(ARC_MNG_ICCM_SIZE + ARC_MNG_DCCM_SIZE)
 
-#define ARC0_ICCM_FULL_BASE		0x300007FE8800000ull	/* rst arc */
-#define ARC0_ICCM_SIZE			SZ_64K
-#define ARC0_DCCM_FULL_BASE		0x300007FE8810000ull
-#define ARC0_DCCM_SIZE			SZ_32K
-#define ARC1_ICCM_FULL_BASE		0x300007FE8840000ull	/* mgmt arc */
-#define ARC1_ICCM_SIZE			SZ_256K
-#define ARC1_DCCM_FULL_BASE		0x300007FE8880000ull
-#define ARC1_DCCM_SIZE			SZ_128K
-#define ARC2_ICCM_FULL_BASE		0x300007FE88C0000ull	/* pid arc */
-#define ARC2_ICCM_SIZE			SZ_128K
-#define ARC2_DCCM_FULL_BASE		0x300007FE88E0000ull
-#define ARC2_DCCM_SIZE			SZ_64K
+#define ARC_RST_ICCM_FULL_BASE		0x300007FE8800000ull
+#define ARC_RST_ICCM_SIZE		SZ_64K
+#define ARC_RST_DCCM_FULL_BASE		0x300007FE8810000ull
+#define ARC_RST_DCCM_SIZE		SZ_32K
+#define ARC_MNG_ICCM_FULL_BASE		0x300007FE8840000ull
+#define ARC_MNG_ICCM_SIZE		SZ_256K
+#define ARC_MNG_DCCM_FULL_BASE		0x300007FE8880000ull
+#define ARC_MNG_DCCM_SIZE		SZ_128K
+#define ARC_PID_ICCM_FULL_BASE		0x300007FE88C0000ull
+#define ARC_PID_ICCM_SIZE		SZ_128K
+#define ARC_PID_DCCM_FULL_BASE		0x300007FE88E0000ull
+#define ARC_PID_DCCM_SIZE		SZ_64K
 
 /* CFG is at Arc region F, specs CFG base address is 0x70000000. 7+8 = 0xF */
 #define HL_CFG_BASE			0x80000000
@@ -44,9 +44,9 @@
 #define PCIE_FW_SRAM_BASE		(FW_MEM_REGION_BASE + 0x380000)
 #define FW_SRAM_BASE			(FW_MEM_REGION_BASE + 0x1A0000)
 #define SP_SRAM_BASE			(FW_MEM_REGION_BASE + 0x200000)
-#define ARC1_ICCM_BASE			(FW_MEM_REGION_BASE + 0x40000)	/* mgmt */
-#define ARC1_DCCM_BASE			(FW_MEM_REGION_BASE + 0x80000)	/* mgmt */
-#define ARC2_ICCM_BASE			(FW_MEM_REGION_BASE + 0xC0000)	/* pid */
+#define ARC_MNG_ICCM_BASE		(FW_MEM_REGION_BASE + 0x40000)
+#define ARC_MNG_DCCM_BASE		(FW_MEM_REGION_BASE + 0x80000)
+#define ARC_PID_ICCM_BASE		(FW_MEM_REGION_BASE + 0xC0000)
 #define HL_SPI_BASE			0x94000000
 
 #define PPBOOT_TEXT_BASE		FW_SRAM_BASE
